scripts/release_scripts/cut_release_or_hotfix_branch.py

```diff
@@ -189,48 +175,94 @@ def _verify_hotfix_number_is_one_ahead_of_previous_hotfix_number(
         'git', 'branch', '-a'])[:-1].split('\n')
 
     last_hotfix_number = 0
+    release_branch_exists = False
     hotfix_branch_name_regex = '^%s/%s-%s-%s-\\d*$' % (
-        remote_alias, release_constants.RELEASE_BRANCH_TYPE, TARGET_VERSION,
+        remote_alias, release_constants.RELEASE_BRANCH_TYPE, target_version,
         release_constants.HOTFIX_BRANCH_TYPE)
     for branch_name in all_branches:
         branch_name = branch_name.lstrip().rstrip()
+        if branch_name == '%s/%s-%s' % (
+                remote_alias, release_constants.RELEASE_BRANCH_TYPE,
+                target_version):
+            release_branch_exists = True
         if re.match(hotfix_branch_name_regex, branch_name):
-            hotfix_number = int(branch_name[branch_name.rfind('-') + 1:])
-            if hotfix_number > last_hotfix_number:
-                last_hotfix_number = hotfix_number
+            branch_hotfix_number = int(branch_name[branch_name.rfind('-') + 1:])
+            if branch_hotfix_number > last_hotfix_number:
+                last_hotfix_number = branch_hotfix_number
+
+    assert release_branch_exists
+    assert hotfix_number == last_hotfix_number + 1
+
+
+def _get_release_branch_type_and_name(target_version):
+    """Returns type and name of release branch for a target version.
+
+    Args:
+        target_version: str. The release version.
+
+    Returns:
+        tuple(str, str). The type and name of release branch.
+    """
+    return (
+        release_constants.RELEASE_BRANCH_TYPE, 'release-%s' % target_version)
```

Comment:
This is following @seanlip's [comment](https://github.com/oppia/oppia/pull/8005#discussion_r351038440) to avoid confusion between type and name. I actually missed updating this at a few places. Fixed now.